io.github.nostar.DroidStar.yml

```diff
@@ -0,0 +1,40 @@
+app-id: io.github.nostar.DroidStar
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: droidstar
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-download
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - "*.a"
+  - "*.la"
+modules:
+  - name: imbe_vocoder
+    buildsystem: simple
+    build-commands:
+      - make
+      - install -Dm644 libimbe_vocoder.a /app/lib/libimbe_vocoder.a
```

Comment:
You also need to install the _imbe_vocoder_api.h_ file into _/app/include_.

Reply:
there is a make install target that should be used instead.

Reply:
ok ascii !! tks Now comes another error .. Let's go apoco! Je
Glad to see you again.. I do not know if you remember but start with Fldigi !

Reply:
Of course I remember you and I am glad to see you back here on GitHub! :-)